crates/hir_def/src/item_tree/lower.rs

```diff
@@ -189,12 +190,16 @@ impl Ctx {
                                 block_stack.push(self.source_ast_id_map.ast_id(&block));
                             },
                             ast::Item(item) => {
-                                // FIXME: This triggers for macro calls in expression/pattern/type position
-                                let mod_items = self.lower_mod_item(&item, true);
-                                let current_block = block_stack.last();
-                                if let (Some(mod_items), Some(block)) = (mod_items, current_block) {
-                                    if !mod_items.0.is_empty() {
-                                        self.data().inner_items.entry(*block).or_default().extend(mod_items.0.iter().copied());
+                                // FIXME: This triggers for macro calls in expression/pattern
+                                if let Some(SyntaxKind::MACRO_TYPE) = node.parent().map(|p| p.kind()) {
```

Comment:
```suggestion
                                if let Some(SyntaxKind::MACRO_TYPE | SyntaxKind::MACRO_PAT) = node.parent().map(|p| p.kind()) {
```

Reply:
@cynecx are you msnduy or duyan if it's the wrong person then sorry to comment on your posts

Reply:
Sorry to disappoint but no, I am not.

Reply:
@cynecx Oki have a great day. Sorry about the posts. I deleted it. Potatao